sections/04_study.md

```diff
@@ -241,114 +241,61 @@ this was easy to simulate with a computational model. As we learn to better
+### Promoters and enhancers
+
+#### From TF binding to promoters and enhancers
+
+Multiple TFs act in concert to coordinate changes in gene regulation at the genomic regions known as promoters and enhancers.
+Each gene has an upstream promoter, essential for initiating that gene's transcription.
+The gene may also interact with multiple enhancers, which can amplify transcription in particular cellular contexts.
+These contexts include different cell types in development or environmental stresses.
+
+Promoters and enhancers provide a nexus where clusters of TFs and binding sites mediate downstream gene regulation, starting with transcription.
+The gold standard to identify an active promoter or enhancer requires demonstrating its ability to affect transcription or other downstream gene products.
+Even extensive biochemical TF binding data has thus far proven insufficient on its own to accurately and comprehensively locate promoters and enhancers.
+We lack sufficient understanding of these elements to derive a mechanistic "promoter code" or "enhancer code".
+But extensive labeled data on promoters and enhancers lends itself to probabilistic classifier.
+The complex interplay of TFs and chromatin leading to the emergent properties of promoter and enhancer activity seems particularly apt for capture by deep neural networks.
+
+#### Promoters
+
+Despite decades of work, computational identification of promoters remains a stubborn problem [@doi:10.1093/bib/4.1.22].
+Researchers have used neural networks for promoter recognition as early as 1996 [@doi:10.1016/S0097-8485(96)80015-5].
+Recently, a CNN recognized promoter sequences with sensitivity and specificity exceeding 90% [@doi:10.1371/journal.pone.0171410].
+Most activity in computational prediction of regulatory regions, however, has moved to enhancer identification.
+Since one can identify promoters with straightforward biochemical assays [@doi:10.1073/pnas.2136655100 @doi:10.1101/gr.110254.110], the direct rewards of promoter prediction alone have decreased.
+But the reliable ground truth provided by these assays makes promoter identification an appealing test bed for deep learning approaches that can also identify enhancers.
+
+#### Enhancers
+
+Recognizing enhancers presents additional challenges.
+Enhancers may be up to 1,000,000 bp away from the affected promoter, and even within introns of other genes [@doi:10.1038/nrg3458].
+Enhancers do not necessarily operate on the nearest gene and may affect multiple genes.
+Their activity is frequently tissue- or context-specific.
+No biochemical assay can reliably identify all enhancers.
+Distinguishing them from other regulatory elements remains difficult, and some believe the distinction somewhat artificial [@doi:10.1016/j.tig.2015.05.007].
+While these factors make the enhancer identification problem more difficult, they also make a solution more valuable.
+
+Several neural network approaches yielded promising results in enhancer prediction.
```

Comment:
This comment probably pertains to some text a bit later in the source:

> A baseline SVM approach [attained] a mean AUPR of 0.322 over 164 cell types `[@doi:10.1101/gr.200535.115]`... DeepEnhancer used a CNN to get a AUPR of 0.917 on nine cell types `[@tag:Min2016_deepenhancer]`, against an AUPR of 0.899 reported for gkmSVM... The dramatic difference in AUPRs reported by two studies for the same method -- gkmSVM -- illustrates the problems in comparing metrics created with different benchmarking procedures.
It's impossible to directly compare the performance of Basset and DeepEnhancer based solely on their reported metrics. DECRES used a different set of metrics altogether. To drive further progress in enhancer identification, we must develop a common and comparable benchmarking procedure (see Discussion).

Generally, I would not want to get into details like AUPR and AUROC numbers. In this particular case, I note that two competing papers are reporting wildly different metrics for the same baseline comparison. This makes it impossible to come up with an opinionated summary of how these methods compare with each other based on the text in the paper, and I think that is an important point to illustrate.

`AUPR` converted to `auPRC` here in michaelmhoffman/deep-review@a425797fa9e983bf445c6b0adde99802a31a6a4c.